test/models/image_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ImageTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
We always want to have meaningful tests for every class we create, let's add some tests here to make sure that an image is valid with a valid url, invalid with an invalid url, and invalid with an empty url.